modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/loading-validate/index.css

```
@@ -0,0 +1,12 @@
+#loading-validate {
+	gap: 24px;
+}
+
+#msg-loading {
+	font-family: 'Source Sans Pro';
+	font-size: 16px;
+	font-style: normal;
+	font-weight: 400;
+	line-height: 24px;
+	text-align: center;
+}
```

Scope the elements by his parent, something like:

```suggestion

#loading-validate {
	gap: 24px;
}

#loading-validate #msg-loading {
	font-family: 'Source Sans Pro';
	font-size: 16px;
	font-style: normal;
	font-weight: 400;
	line-height: 24px;
	text-align: center;
}
```

Does is really necessary to set here "font-family", "font-size", "font-style", "font-weight"... here? it seems copy-paste from Figma (which is not a problem) but Figma generates those styles for all elements, which many of then are already set globally

If possible, keep the attributes needed here